pages/coinbase/homepage.py

```diff
@@ -30,27 +40,242 @@ def __init__(self, driver, spawn=True):
     company_menu_loc =      (by.XPATH, "//button[contains(text(),'Company')]")
     earn_crypto_link_loc =  (by.XPATH, "//a[@title='Earn crypto']")
     sign_in_link_loc =      (by.XPATH, "//a[@title='Sign in']")
-    header_get_started_button_loc =(
-        by.XPATH, "//a[.='Get started']")
+    header_get_started_button_loc = (by.XPATH, "//a[.='Get started']")
+
+         
+    # CREATE ACCOUNT
+
+    email_address_field_loc = (by.XPATH, "//input[@type='email']")
+    get_started_button_loc =  (by.XPATH,
+                            "//input[@type='email']/following-sibling::button")
```

Comment:
Your indentation here doesn't match your indentation elsewhere.

Reply:
for line 50 this is indented as far as it can to fit in the 79 character range